layouts/docs/single.html

```diff
@@ -1,4 +1,26 @@
 {{ define "main" }}
+
```

Comment:
This includes MathJax in all RFCs.  It attempts to not parse the values of `skipTags`.

It relies on a CDN for which we should either a) host ourselves, or b) declare a checksum for the contents.